src/main/resources/blogs/companies.json

```diff
@@ -109,6 +109,12 @@
       "name": "Smart Up",
       "rss": "https://up.smartrecruiters.com/feed",
       "twitter": "@SmartRecruiters"
+    },
+    {
+      "bookmarkableId": "detektywi.it",
```

Comment:
As you have probably already noticed `bookmarkableId` cannot contain `.`. It must conform to `[a-z0-9-]`

Reply:
Fixed